admob/src/include/firebase/admob.h

```diff
@@ -45,13 +45,20 @@ namespace admob {
 
 /// Initializes AdMob via Firebase.
 ///
-/// @param app The Firebase app for which to initialize mobile ads.
+/// @param[in] app The Firebase app for which to initialize mobile ads.
 ///
-/// @return kInitResultSuccess if initialization succeeded, or
+/// @param[out] init_result_out Optional: If provided, write the basic init
+/// result here. kInitResultSuccess if initialization succeeded, or
 /// kInitResultFailedMissingDependency on Android if Google Play services is not
 /// available on the current device and the Google Mobile Ads SDK requires
 /// Google Play services (for example, when using 'play-services-ads-lite').
```

Comment:
Perhaps add a note that if this isn't a success value then returned future is invalid?